src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+/* Styles go here  */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  height: 40px;
+  margin: 10px 0;
+}
+
+.header__navbar {
+  align-items: center;
+}
+
+.header__menu {
+  list-style: none;
+  display: flex;
+  max-height: 60px;
+  padding: 18.5px 0;
```

```suggestion
  padding:  0;
```
don't set paddings, use display inline block on links instead